autovalet_navigation/src/cmd_vel_listener.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
```

Comment:
what did we use this node for? i dont even remember

Reply:
you were trying to solve some planner issue by observing the velocity in real time? I remember you and Subbu were trying to tune the planner yaml files when you had this code written.

Reply:
the initial commit for the file is from Kevin. I don't remember writing the file because quaternion_from_matrix sounds very new to my brain. I think @kob51 you prolly wrote it sometime to debug the nav stack. It's good to let it be I guess.

Reply:
will keep it then.

Reply:
lol cool